lua/obsidian/commands/open.lua

```diff
@@ -81,5 +82,13 @@ return function(client, data)
 
   assert(cmd)
   assert(args)
-  run_job(cmd, args)
+
+  vim.fn.jobstart(cmd .. " " .. table.concat(args, " "), {
+    detach = true,
+    on_exit = function(_, exit_code)
+      if exit_code ~= 0 then
+        log.err("open command's job failed with exit code '" .. exit_code .. "'")
```

Comment:
Could you also log the final cmd string that was passed to `jobstart` when it fails?